test/FastTests/Client/Indexing/JavaScriptIndexTests.cs

```diff
@@ -665,7 +665,7 @@ public UsersByNameAndAnalyzedName()
                             @"map('Users', function (u){
                                     return {
                                         Name: u.Name,
-                                        _: {$value: u.Name, $name:'AnalyzedName', $options:{index: true, store: true}}
+                                        _: {$value: u.Name, $name:'AnalyzedName', $options:{indexing: 'Search', storage: true}}
```

Comment:
isn't this a breaking change?
This is experimental feature, though.

Reply:
This was my thinking. We can support both if needed but I do not think that this is a good idea in this case. Can you make a call here?

Reply:
It's fine to make this change, yes